### PR TITLE
feat(theme): replace hard coded colors

### LINKED ISSUE #416: feat(theme): replace hard coded colors
There are some places where the colors are hard coded. Replace those colors.
Eg:-
Screenshot from 2023-02-05 02-34-46
Screenshot from 2023-02-05 02-36-33

For Reference :- #415

### PR DESCRIPTION
**Pull request by Anish-Sarawgi**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #416 

### Description

image

image

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Review by Shurtu-gal — APPROVED** [review]

File: client/src/components/photostory/Carousel.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ const useStyles = makeStyles((theme) => ({
   wrapper: {
     maxWidth: '100%',
     position: 'relative',
-    backgroundColor: theme.palette.secondary.main,
+    backgroundColor: '#222222',
```
Comment (120EE0692) [inline client/src/components/photostory/Carousel.js:118]: @Anish-Sarawgi is this fine for both light and dark theme?

File: client/src/components/widgets/BigCarousel.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default BigCarousel;
 const useStyles = makeStyles((theme) => ({
   wrapper: {
     marginTop: '2rem',
-    backgroundColor: theme.palette.secondary.main,
+    backgroundColor: '#222222',
```
Comment (120EE0692) [inline client/src/components/widgets/BigCarousel.js:27]: @Anish-Sarawgi is this fine for both light and dark theme?

File: client/src/components/photostory/Carousel.js (hunk 1 above)
Comment (Anish-Sarawgi) [reply client/src/components/photostory/Carousel.js:118]: Yes it is fine. As the background color for this component will be same in both light and dark mode, this way works.

File: client/src/components/widgets/BigCarousel.js (hunk 2 above)
Comment (Anish-Sarawgi) [reply client/src/components/widgets/BigCarousel.js:27]: Yes.

**Review by 120EE0692 — APPROVED** [review]

**Pull request merged**